specificatie/gba-genereervariant/openapi.json

```diff
@@ -297,38 +297,47 @@
   },
   "components" : {
     "schemas" : {
+      "Reisdocumentnummer" : {
+        "pattern" : "^[a-zA-Z0-9]{9}$",
+        "type" : "string"
+      },
       "GbaReisdocument" : {
         "type" : "object",
         "properties" : {
-          "reisdocumentnummer" : {
-            "$ref" : "#/components/schemas/Reisdocumentnummer"
-          },
-          "soortReisdocument" : {
-            "$ref" : "#/components/schemas/Waardetabel"
-          },
-          "autoriteitAfgifte" : {
-            "$ref" : "#/components/schemas/Waardetabel"
-          },
-          "datumUitgifte" : {
-            "$ref" : "#/components/schemas/GbaDatum"
-          },
           "datumEindeGeldigheid" : {
             "$ref" : "#/components/schemas/GbaDatum"
           },
           "datumInhoudingOfVermissing" : {
```

Comment:
Heb je deze property wel nodig om in de HC variant van deze API te komen tot 'indicatieGeldig'? 
Mij lijkt dat je alleen de properties
* datumEindeGeldigheid
* aanduidingInhoudingOfVermissing
* overlijden

nodig hebt om die property van een waarde te kunnen voorzien.

Reply:
alleen voor de zekerheid, maar wellicht dubbelop: 
Als aanduidingInhoudingOfVermissing een waarde heeft Of datumInhoudingOfVermissing een waarde heeft, Dan is het reisdocument ongeldig.